lisa/operating_system.py

```diff
@@ -1562,6 +1562,7 @@ def _get_package_information(self, package_name: str) -> LisaVersionInfo:
         )
         return self._cache_and_return_version_info(package_name, version_info)
 
+    @retry(tries=10, delay=5)
```

Comment:
Please use the original numbers `tries=3, delay=1`. I'm worry about it may extend the runtime too long, if something failed continually.